src/commands/orders.py

```diff
@@ -226,8 +229,93 @@ def print_orders_pretty(orders):
     )
 
 def print_orders_csv(orders):
-  # TODO: Implement here the CSV formatting
-  click.echo("Not implemented yet")
+
+  writer = get_csv_writer()
+
+  writer.writerow([
+    'ID',
+    'From Batch Id',
+    'Until Batch Id',
+    'Create Date',
+    'Cancel Date',
+    'Delete Date',
+    'Pair Sell/Buy',
+    'Limit Price Sell/Buy',
+    'AVG Price Sell/Buy',
+    'Pair Buy/Sell',
+    'Limit Price Buy/Sell',
+    'AVG Price Buy/Sell',
+    'Sell Token',
+    'Sold Volume',
+    'Max Sell Amount',
+    'Buy Token',
+    'Bought Volume',
+    'Timestamp',
+    'Transaction',
+    'Trader Address',
+    ])
+
+  for order in orders:
+
+    price_sell_buy = unformat_amount(format_price(
+      calculate_price(
+          numerator=order['price_numerator'],
+          denominator=order['price_denominator'],
+          decimals_numerator=order['buy_token']['decimals'],
+          decimals_denominator=order['sell_token']['decimals'],
+      )))
+
+    price_buy_sell = unformat_amount(format_price(
```

Comment:
Why do you format for later unformat, it doesn't sound right?

Reply:
Fixed this one.